src/main/webapp/vendor/startbootstrap/index.html

```diff
@@ -18,6 +18,11 @@
   <!-- Custom styles for this template-->
   <link href="https://cdn.jsdelivr.net/npm/startbootstrap-sb-admin-2@4.1.1/css/sb-admin-2.min.css" rel="stylesheet">
 
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="https://uicdn.toast.com/tui.time-picker/latest/tui-time-picker.css">
+  <link rel="stylesheet" type="text/css" href="https://uicdn.toast.com/tui.date-picker/latest/tui-date-picker.css">
+  <link rel="stylesheet" type="text/css" href="https://uicdn.toast.com/tui-calendar/latest/tui-calendar.css" />
+  <link rel="stylesheet" type="text/css" href="../toast/css/style.css">
```

Comment:
This should be  
```suggestion
  <link rel="stylesheet" type="text/css" href="vendor/toast/css/style.css">
```
because its being copied into the webapp folder